Automattic/jetpack
# AI Assistant: Remove unused code

Fixes #

## Proposed changes:

* Removes unused code related to prompting
* Fixes some typescript issues

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






The features should not change, so:

* Use the AI Assistant and its features
* Check that there is no regression

Features to test:
* AI Assistant block
* Toolbar options (translate, fix grammar, etc.)
* Extension (paragraph, heading, list)
* Form
* Excerpt
* Feedback
